api/v1/openapi.yaml

```
@@ -1834,6 +1837,22 @@ definitions:
         type: array
         items:
           type: string
+  SRv6:
+    description: |-
+      Status of the host srv6
+
+      +k8s:deepcopy-gen=true
+    type: object
+    properties:
+      mode:
+        type: string
+        enum:
+        - Disabled
+        - Enabled
```

I'm not quite sure why I used an enum for the host firewall, but it can be a boolean here:
```suggestion
      enabled:
        type: boolean
```